keystorage/hashicorp/src/test/java/tech/pegasys/signers/hashicorp/MockedVertxHashicorpConnectionFactoryTest.java

```diff
@@ -24,84 +22,24 @@
 import java.io.IOException;
 import java.util.Optional;
 
-import io.vertx.core.Vertx;
-import io.vertx.core.http.HttpClientOptions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
-import org.mockito.ArgumentCaptor;
 
 class MockedVertxHashicorpConnectionFactoryTest {
```

Comment:
Test should be renamed now since we no longer need Vertx

Reply:
good catch, merged tests of this class to HashicorpConnectionFactoryTest.